src/app/core/facades/theme.facade.ts

```diff
@@ -0,0 +1,58 @@
+import { DOCUMENT } from '@angular/common';
+import { Inject, Injectable, Renderer2, RendererFactory2 } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { BehaviorSubject } from 'rxjs';
+import { distinctUntilChanged } from 'rxjs/operators';
+
+import { THEMES } from 'ish-core/configurations/injection-keys';
+import { Theme } from 'ish-core/models/theme/theme.model';
+import { getTheme } from 'ish-core/store/configuration';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ThemeFacade {
+  theme$: BehaviorSubject<string>;
+
+  private renderer: Renderer2;
+  private head: HTMLElement;
+  private themeLinks: HTMLElement[] = [];
+
+  constructor(
+    private rendererFactory: RendererFactory2,
+    @Inject(DOCUMENT) private document: Document,
+    @Inject(THEMES) private themes: Theme[],
+    store: Store<{}>
+  ) {
+    this.head = this.document.head;
+    this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
+    this.theme$ = new BehaviorSubject(this.themes.find(x => x.default).name);
```

Comment:
how about initializing the `theme$` with `new ReplaySubject(1)` and just `next`ing the first value here?